web3.go

```diff
@@ -336,3 +375,81 @@ func ParseLogs(myabi abi.ABI, logs []*types.Log) ([]Event, error) {
 	}
 	return output, nil
 }
+
+func ParseAmount(amount string) (*big.Int, error) {
```

Comment:
Should document that it returns wei/atto

Reply:
We probably need to support decimals, e.g. `0.5go`. Otherwise users will have to convert to smaller units to get integers, and we aren't even supporting all the increments currently, so `0.5go` would have to be `500000000nano`.